src/main/java/application/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,39 @@
 package application.onlinebookstore;
 
+import application.onlinebookstore.model.Book;
+import application.onlinebookstore.service.BookService;
+import java.math.BigDecimal;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
Don't forget about dependency injection - provide this service through constructor and add @ Autowired above constructor.